e2e/pageDetail/transactions/accountAddressRestriction.spec.js

```diff
@@ -15,7 +16,7 @@ describe('Symbol Explorer Transaction detail page for Account Address Restrictio
         })
 
         it('render correct transaction info titles', ()=> {
-            const items = ['Block Height', 'Transaction Hash', 'Transaction ID', 'Date', 'Deadline', 'Fee', 'Signature', 'Signer', 'Status', 'Confirmation']
+            const items = datafiled.transactionInfoField
```

Comment:
`transactionInfoField` maybe `transactionInfoFields` if there is an array?

Reply:
nice catch!